xbmc/network/DNSNameCache.cpp

```diff
@@ -22,35 +22,60 @@
 #include "threads/SingleLock.h"
 #include "utils/log.h"
 #include "utils/StringUtils.h"
+#include "network/Network.h"
+#include "Application.h"
 
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <netdb.h>
 
+using namespace ANNOUNCEMENT;
+using namespace std;
```

Comment:
I saw that you used the explicite std:: scope in this diff already. This is also the preferred thing to do - so you can get rid of the using namespace std please ;)